jobs/transforms/user_audits.py

```diff
@@ -52,6 +52,34 @@ def __init__(self):
         self._pattern = feconf.USER_ID_REGEX
 
 
+@audit_decorators.AuditsExisting(user_models.PendingDeletionRequestModel)
+class ValidateActivityMappingOnlyAllowedKeys(beam.DoFn):
+    """DoFn to check for Validates that pseudonymizable_entity_mappings"""
```

Comment:
Add full stop at the end of docstring. It should have been captured in the lint checks right?

Reply:
done!. I think sometimes it is not identifying this error , if you see on the line `85` the same error is already present. Thanks!

Reply:
Then change it there also, and please raise an issue regarding this lint error.

Reply:
Sure